nix/supervisord-cluster/default.nix

```diff
@@ -336,7 +336,7 @@ let
     ${pkgs.python3Packages.supervisor}/bin/supervisord --config ${supervisorConfig} $@
     while [ ! -S $CARDANO_NODE_SOCKET_PATH ]; do echo "Waiting 5 seconds for bft node to start"; sleep 5; done
     echo "Transfering genesis funds to pool owners, register pools and delegations"
-    cardano-cli shelley transaction submit \
+    cardano-cli shelley transaction submit --shelley-mode \
```

Comment:
Does our test cluster run in Shelley only mode?

Reply:
@disassembler can you please help us to merge this PR? The scripts need this in order to provide accurate results. Thank you.